packages/shared/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/shared",
-  "version": "0.1.10",
+  "version": "0.1.11-alpha.1",
```

Comment:
`shared` doesn't depend on the SDK, so in theory didn't need to be updated.

when the change to packages are limited to a package or two, it's better to release the individual package - [guide](https://www.notion.so/metriport/Cheat-Sheet-fd9439faae1944d5b5da861f0e07801e?pvs=4#3134e8b922ca4cffb26240a584ee396f)